src/services/ffmpegService.ts

```diff
@@ -56,6 +56,38 @@ class FFmpegService {
             }
         }
     }
+
+    async convertLivePhotoToMP4(file: Uint8Array): Promise<Uint8Array> {
+        if (!this.ffmpeg) {
+            await this.init();
+        }
+        if (this.isLoading) {
+            await this.isLoading;
+        }
+
+        try {
+            this.ffmpeg.FS('writeFile', 'input.mov', file);
+            console.log('starting encoding', new Date().toLocaleTimeString());
+            await this.ffmpeg.run(
+                '-i',
+                'input.mov',
```

Comment:
try to keep the code generic so that we could reuse it to convert any type of file to any required format